packages/playwright/src/index.ts

```diff
@@ -147,7 +147,14 @@ export default class AxeBuilder {
       context
     );
     const partials = await partialResults.getPartials();
-    return this.finishRun(partials);
+
+    try {
+      return await this.finishRun(partials);
+    } catch (error) {
+      throw new Error(
+        `Error: ${error}\n Please check out https://github.com/dequelabs/axe-core-npm/blob/develop/packages/playwright/error-handling.md`
```

Comment:
This won't serialise properly. If you want just the message use error.message, or if you want the stack trace use error.trace.

Reply:
`error.trace` is not a standard property. Do not use that.